pallets/assets/src/lib.rs

```diff
@@ -681,7 +681,7 @@ pub mod pallet {
 		///
 		/// Weight: `O(1)`
 		#[pallet::weight(T::WeightInfo::freeze_asset())]
-		pub(super) fn freeze_asset(
+		pub fn freeze_asset(
```

Comment:
this change may impact on the VCU bond. Please check with Ayush. I agree to remove the "super" definition.

Reply:
I'm consulting Ayush on these two then

Reply:
ok